tests/unit/test_flist.py

```diff
@@ -2,11 +2,12 @@
 import sys
 import os
 import re
-import random 
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../fpu')))  # noqa
-from fp import *  # noqa
-import flist
-from flist import *  # noqa
+import random
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../')))  # noqa
```

Comment:
I have tested the code from my local, and humbly think that we don't need the `sys.path.insert` line.

Reply:
That's right. Fixed by merging to latest master branch.